docs/source/feeds.rst

```diff
@@ -26,6 +26,14 @@ Shows user stream for currently logged in user.
     /activity/feed/atom/
     /activity/feed/json/
 
+If you want to include the user own activity, the optional parameter ``with_user_activity`` can be passed to the user stream as a query string parameter:
```

Comment:
I am willing to know if this is part of activity stream specification?

Reply:
meh